charts/akeyless-secure-remote-access/values.yaml

```diff
@@ -99,7 +99,10 @@ ztbConfig:
     # tag: latest
     
   containerName: "zero-trust-bastion"
-  env: []
+  env:
+    # This env variable instruct docker to start only web-bastion processes
+    - name: "BASTION"
```

Comment:
why not BASTION_TYPE ?
BASTION is a bit confusing name

Reply:
dont add this to env part.. put it in a separate section

Reply:
done